common/source/docs/common-serial-led-neopixel.rst

```diff
@@ -5,11 +5,15 @@ NeoPixel
 
 .. image:: ../../../images/neopixel-string.jpeg
 
-A "NeoPixel" style (WS2812B compatible) RGB LED can be attached to any PWM output by setting its SERVOx_FUNCTION to one of the ``NeoPixelx`` output functions and setting :ref:`NTF_LED_TYPES<NTF_LED_TYPES>` parameter to ``NeoPixel``. Multiple ``NeoPixelx`` output functions are provided for connecting multiple strings (up to 4).
+Up to four "NeoPixel" style (WS2812B compatible) RGB LEDs can be used to display the same LED patterns as other RGB LEDs including those commonly installed on GPS/compass units
 
-If used for notification purposes, be sure to set :ref:`NTF_LED_TYPES<NTF_LED_TYPES>` "Neopixel" bit(8).
+Attach each LED to a separate PWM output and set the following parameters
 
-.. note:: a few "NeoPixel" LED types have a different data order for the red and green data instead of the normal green/red/blue orde. If you are not displaying the desired colors, try setting the ref:`NTF_LED_TYPES<NTF_LED_TYPES>` parameter to ``NeoPixelRGB`` tpe instead.
+- Set :ref:`NTF_LED_TYPES<NTF_LED_TYPES>` to include ``NeoPixel`` (bit 8)
+- Set :ref:`NTF_LED_LEN<NTF_LED_LEN>` to the number of LEDs connected (up to 4)
```

Comment:
No this is incorrect. The 4 refers to the channels in SERVOx_FUNCTION - there are four possibilities. NTF_LED_LEN can be 128 if you want (might not work), lengths of 44 have been tested successfully

Reply:
@andyp1per,

OK, how about,

`- Set :ref:`NTF_LED_LEN<NTF_LED_LEN>` to the number of PWM outputs configured (up to 4).  E.g the number of strings of LEDs connected.`

Reply:
Set NTF_LED_LEN to the number of LEDs connected to a servo output

Reply:
@andyp1per,

OK, so this number is the number of LEDs on the longest string?  .. or perhaps the total number of LEDs across all strings?  .. I guess it is not the number of strings.

Reply:
ok updated to:

- Set NTF_LED_LEN to the number of LEDs connected to a servo output (up to 4)